server/datastore/mysql/software_titles.go

```diff
@@ -202,7 +202,8 @@ SELECT
 	st.browser,
 	MAX(COALESCE(sthc.hosts_count, 0)) as hosts_count,
 	MAX(COALESCE(sthc.updated_at, date('0001-01-01 00:00:00'))) as counts_updated_at,
-	si.filename as software_package
+	si.filename as software_package,
+	MAX(COALESCE(si.self_service, false)) as self_service
```

Comment:
I think it shouldn't use an aggregation function just the COALESCE, and be added to the GROUP BY clause (a software title results in a single row, and a software installer is 1:1 with a software title).

Reply:
Ok, will change

Reply:
I presume this is also handled by your branch as well?

Reply:
I didn't change the `SELECT` list, just the conditions/filter.